### PR TITLE
Home button implementation

### PR DESCRIPTION
**Pull request by jilliebean**
With this commit, I added home button functionality to the events and map
page. Furthermore, I changed the home page paragraph to something more
relevant to the functionality of our website.

### CODE REVIEW

**Comment by brforest** [comment]
Definitely like the new text on the home page.  However, I don't think that we should add the `/home/ urls`.  The home page should just be at the root of the web directory, just `/`.  All that would require is just removing the added line in `dontatefood/urls.py`, and changing the button link to `/` (instead of `/home/`).

**Comment by mfraser4** [comment]
I agree with @brforest 's comments.  Also, the conflicts on this branch will have to be resolved with a rebase (`git pull --rebase origin master`) before this can be merged.

File: donatefood/home/templates/home/home.html [hunk 1]
```diff
@@ -48,12 +48,15 @@ <h1 class="w3-margin w3-jumbo">Food Bank Finder</h1>
...
-        <h5 class="w3-padding-32">Insert sentimentally-convincing statement here</h5>
+        <h5 class="w3-padding-32">Giving is not just about making a donation, it's about making a difference</h5>
         <div class="w3-twothird">
-          <h1>Lorem Ipsum</h1>
-          <h5 class="w3-padding-32">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor
-            incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco
-            laboris nisi ut aliquip ex ea commodo consequat.</h5>
+          <h1>Why Donate?</h1>
+          <h5 class="w3-padding-32">
+Donating to charity is a major mood-booster. The knowledge that you’re helping others is hugely empowering and, in turn, can make you feel happier and more fulfilled. Research has identified a link between making a donation to charity and increased activity in the area of the brain that registers pleasure - proving that as the old adage goes, it really is far better to give than to receive.
```
Comment (mfraser4) [inline donatefood/home/templates/home/home.html]: proving that, as the old adage goes

The above statement should have a comma.

File: donatefood/home/templates/home/home.html [hunk 2]
```diff
@@ -48,12 +48,15 @@ <h1 class="w3-margin w3-jumbo">Food Bank Finder</h1>
...
+
+Our own research into why people give supports this. We asked 700 of our generous donors to tell us what motivates them to give regularly to charity; 42% agreed the enjoyment they receive from giving as a key influence.
```
Comment (mfraser4) [inline donatefood/home/templates/home/home.html]: We have not done this and should not advertise falsely, especially if we release this on the internet.

**Comment by mfraser4** [comment]
I've also added a couple comments of my own.  Please let me know what you think when you get the chance.

**Comment by brforest** [comment]
Looks good!  I'm good with merging this.

**Pull request merged**